### PR TITLE
Refreshing with the latest from office-js-snippets

### PR DESCRIPTION
**Pull request by AlexJerabek**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[fa310d5](https://github.com/OfficeDev/office-js-docs-reference/commits/fa310d56dc23526d6c858d4a6126bd53c93282d6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/docs-ref-autogen/excel/excel/excel.range.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/docs/docs-ref-autogen/excel/excel/excel.range.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.workbook.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/docs/docs-ref-autogen/excel/excel/excel.workbook.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_7/excel/excel.workbook.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/docs/docs-ref-autogen/excel_1_7/excel/excel.workbook.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_8/excel/excel.workbook.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/docs/docs-ref-autogen/excel_1_8/excel/excel.workbook.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_9/excel/excel.range.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/docs/docs-ref-autogen/excel_1_9/excel/excel.range.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_9/excel/excel.workbook.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/docs/docs-ref-autogen/excel_1_9/excel/excel.workbook.yml) | :white_check_mark:Succeeded | |
[generate-docs/json/excel/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/generate-docs/json/excel/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/excel_1_1/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/generate-docs/json/excel_1_1/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/excel_1_2/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/generate-docs/json/excel_1_2/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/excel_1_3/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/generate-docs/json/excel_1_3/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/excel_1_4/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/generate-docs/json/excel_1_4/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/excel_1_5/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/generate-docs/json/excel_1_5/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/excel_1_6/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/generate-docs/json/excel_1_6/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/excel_1_7/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/generate-docs/json/excel_1_7/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/excel_1_8/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/generate-docs/json/excel_1_8/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/excel_1_9/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/generate-docs/json/excel_1_9/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/generate-docs/json/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/script-lab-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/generate-docs/script-inputs/script-lab-snippets.yaml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C24%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C201909241732582595-515%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=dbBMqCN1wzqNMZAaH9IblbDZ21pgcBRfylDGsiXzCx8%3D&st=2019-09-24T17%3A47%3A15Z&se=2019-10-25T17%3A52%3A15Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: generate-docs/json/excel_1_2/snippets.yaml [hunk 1]
```diff
@@ -3568,6 +3594,23 @@
...
+
+        // This collapses the entire outline.
+        sheet.showOutlineLevels(1, 1);
+        await context.sync();
+    });
+  - |-
+    Excel.run(async (context) => {
+        const sheet = context.workbook.worksheets.getActiveWorksheet();
+
+        // This shows the top 3 outline levels; collapsing any additional sublevels
```
Comment (ElizabethSamuel-MSFT) [inline generate-docs/json/excel_1_2/snippets.yaml]: small thing: should end with a period
```suggestion
        // This shows the top 3 outline levels; collapsing any additional sublevels.
```

Comment (AlexJerabek) [reply generate-docs/json/excel_1_2/snippets.yaml]: Good catch. I'll update it in the source.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[79b0612](https://github.com/OfficeDev/office-js-docs-reference/commits/79b0612541ff315655d1fb4eea2723c14d7aa4a0)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/docs-ref-autogen/excel/excel/excel.range.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/docs/docs-ref-autogen/excel/excel/excel.range.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.workbook.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/docs/docs-ref-autogen/excel/excel/excel.workbook.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_7/excel/excel.workbook.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/docs/docs-ref-autogen/excel_1_7/excel/excel.workbook.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_8/excel/excel.workbook.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/docs/docs-ref-autogen/excel_1_8/excel/excel.workbook.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_9/excel/excel.range.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/docs/docs-ref-autogen/excel_1_9/excel/excel.range.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_9/excel/excel.workbook.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/docs/docs-ref-autogen/excel_1_9/excel/excel.workbook.yml) | :white_check_mark:Succeeded | |
[generate-docs/json/excel/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/generate-docs/json/excel/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/excel_1_1/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/generate-docs/json/excel_1_1/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/excel_1_2/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/generate-docs/json/excel_1_2/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/excel_1_3/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/generate-docs/json/excel_1_3/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/excel_1_4/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/generate-docs/json/excel_1_4/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/excel_1_5/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/generate-docs/json/excel_1_5/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/excel_1_6/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/generate-docs/json/excel_1_6/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/excel_1_7/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/generate-docs/json/excel_1_7/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/excel_1_8/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/generate-docs/json/excel_1_8/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/excel_1_9/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/generate-docs/json/excel_1_9/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/json/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/generate-docs/json/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/script-lab-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-Refresh/generate-docs/script-inputs/script-lab-snippets.yaml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C24%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C201909242018313579-515%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=WSs3bHgw7IeDlsWmr2walEix69jdMNrS0n52hl1CiuI%3D&st=2019-09-24T20%3A33%3A27Z&se=2019-10-25T20%3A38%3A27Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**